config.js

```diff
@@ -512,15 +500,6 @@ let enabledChannels = {
       offline: "FeelsBadMan Ebbel has gone offline FeelsBadMan 👉 ",
     },
   },
-  echoflex: {
-    id: 128217169,
-    formats: {
-      title: "PagChomp NEW TITLE! PagChomp 👉 $VALUE$ 👉 ",
-      game: "PagChomp NEW GAME! PagChomp 👉 $VALUE$ 👉 ",
-      live: "PagChomp 👉 EchoFlex has gone live PagChomp 👉 ",
-      offline: "FeelsBadMan EchoFlex has gone offline FeelsBadMan 👉 ",
-    },
-  },
```

Comment:
![image](https://user-images.githubusercontent.com/41973452/127143022-e5b5be30-cfe3-4eac-941c-ce5d8f80d912.png)

Reply:
Yeah I think he wants 2 bots for some reason, oh well